### ISSUE TITLE
"Prove your website, with DNS" instructions seem to be wrong

### ISSUE THREAD

**Issue by JulesBrookfield** [keybase/client#10226]
It seems to me that the instructions on keybase.io website and windows app say to add a TXT record where the name is blank or "@".  I did this, but my website/domain verification would always fail.  Once I took a look at what the command line was expecting, which seemed to be "_keybase", I changed the name to "_keybase" and it worked wonderfully - I've tried this with 3 more domains and the process and resolution was the same.

**Comment by raiderrobert** [comment]
Agreed. Also, if you're setting up SPF records for your domain (which you should do!), this makes it difficult to merge the records. 

The instructions should be "_keybase" imo. Below is my rewritten copy.

> Enter the following as a TXT entry in your DNS zone, exactly as it appears. If you need a "name" for an entry, use "_keybase".

<img width="528" alt="Screen Shot 2019-12-13 at 8 52 17 AM" src="https://user-images.githubusercontent.com/806571/70805014-f2d32080-1d85-11ea-9af4-ecdd6afc0301.png">
